about.html

```diff
@@ -151,10 +151,10 @@ <h3>Simple Budgeting</h3>
   <section class="partner" id="footer">
     <h3>Partner</h3> <hr>
     <ul>
-      <li>Bank-Central</li>
```

Comment:
:red_circle: I appreciate your extra effort,  but remember the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md) say you should follow the design guidelines. Your about-page layout looks different from the one provided. 

![requirement](https://github.com/aradradev/easyBankTC/assets/26736582/bf03a6ff-2fe6-4347-8e3f-19b4ff7f5744)

I can't see any section similar to this on your About page
![design](https://github.com/aradradev/easyBankTC/assets/26736582/155b93e3-e62f-4acb-a93f-3aa7befe7d39)

Please try to follow the page layout as closely as possible